lib/services/install.js

```diff
@@ -0,0 +1,169 @@
+var q = require('q');
+var fs = require('fs');
+var path = require('path');
+var shell = require('shelljs');
+var prompt = require('prompt');
+var colors = require('colors/safe');
+
+/**
+ * Installs instances of PencilBlue.
+ * @constructor
+ */
+var InstallService = function() {};
+
+/**
+ * Installs the application with version and location settings.
+ *
+ * @param  {String} [version]   The version of PencilBlue to install.
+ * @param  {String} [directory] The directory to install PencilBlue to.
+ * @return {Promise} A promise that resolves when installation is complete.
+ */
+InstallService.prototype.installWithSettings = function(version, directory) {
+  var deferred = q.defer();
+  var self = this;
+
+  self.confirmRequirements().then(function() {
```

Comment:
I put it there for any potential, future async needs, but I'll just have it return a boolean for now.